assets/js/blocks/active-filters/types.ts

```diff
@@ -4,3 +4,5 @@ export interface Attributes {
 	displayStyle: string;
 	className?: string;
 }
+
+export type IsEditor = boolean;
```

Comment:
We should keep `isEditor` inside the other type definitions and use camel case for the name. 

```suggestion
	className?: string;
	isEditor?: boolean;
}
```

Reply:
I was naming it `isEditor` earlier but then I was getting this error `48:2  error  'isEditor' is already declared in the upper scope on line 36 column 22  @typescript-eslint/no-shadow`

Reply:
When defining an interface or a type, the name needs to start with a captal letter. Thus, `export interface Attributes ...` and `export type IsEditor ...` are valid named. What I meant with my suggestion above, was not to rename the type definition, but to move `isEditor?: boolean;` inside `export interface Attributes` and rename that interface from `Attributes` to `ActiveFiltersBlockProps`.